pkg/operator/ceph/disruption/clusterdisruption/osd.go

```diff
@@ -42,25 +40,83 @@ import (
 
 const (
 	// PDBAppName is that app label value for pdbs targeting osds
-	PDBAppName         = "rook-ceph-osd-pdb"
-	disabledPDBKey     = "disabled-pdb"
-	disabledPDBTimeKey = "pdb-disabled-at"
+	PDBAppName            = "rook-ceph-osd-pdb"
```

Comment:
The type of resource is a pdb, so we really don't need it in the name
```suggestion
	osdPDBAppName            = "rook-ceph-osd"
```

Reply:
updated the name.